### DISCUSSION TITLE
Signature Interface

### DISCUSSION THREAD

**Discussion by Attacler** [directus/directus#5213] [Feature Requests]
Hello everyone!

Basicly i got the idea to add a simple signature field, which basicly means that you have a input field that accepts drawings in them.
basicly something like: http://szimek.github.io/signature_pad/ but inside a field
Then whenever there was input delivered and you save the record, it will be saved as an image.
This could come in handy when you have contracts or similar stuff.

**Comment by rijkvanzanten** [comment]
This sounds like a perfect custom interface to me (especially with the upcoming https://directus.market in mind)

**Comment by benhaynes** [comment]
I like it. That library is MIT, so we can definitely look into rolling this into Directus Core as an Interface. If it doesn't pass 80/20, then it's a great candidate for the Marketplace!

**Reply by Attacler** [reply]
Alright! thank you, if it wont be implemented, ill make something work with an extension

**Reply by benhaynes** [reply]
We (the Core team) probably won't have time to build this in the next few months, so I'd recommend creating an extension based on that library. I'd love to check it out once created!

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
